src/main/java/core/basesyntax/Enum.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Enum {
```

Comment:
The name of the class `Enum` is not descriptive and can be confusing because `Enum` is a reserved keyword in Java. Consider renaming the class to something more specific, like `ColorEnum` to avoid confusion and better represent the content.